play.rb

```diff
@@ -0,0 +1,100 @@
+$LOAD_PATH << '.'
+require 'deck'
+require 'dealer'
+require 'player'
+
+def check_bust(player, dealer)
+  if player.total > 21
+    puts 'PLAYER BUST!! Dealer wins! '\
+    "Player total: #{player.total}, Dealer total: #{dealer.total}"
+    end_game
+  end
+  if dealer.total > 21
+    puts 'DEALER BUST!! Player wins! '\
+    "Player total: #{player.total}, Dealer total: #{dealer.total}"
+    end_game
+  end
+end
+
+def check_win(player, dealer)
+  d = 21 - dealer.total # How close dealer total is to 21
+  pl = 21 - player.total # How close player total is to 21
+  puts 'TIE GAME!' if d == pl
+  puts 'DEALER WINS! '\
+  "Player total: #{player.total}, Dealer total: #{dealer.total}" if d < pl
+  puts 'PLAYER WINS! '\
+  "Player total: #{player.total}, Dealer total: #{dealer.total}" if d > pl
+end
+
+def end_game
+  puts 'Thanks for playing! Seeya next time'
+  exit!
+end
+
+# Game starts here...
+puts 'Welcome to Blackjack!'
+deck = Deck.new
+
+p1 = Player.new
+p1.hand = [deck.game_deck[0], deck.game_deck[1]]
```

Comment:
I alluded to methods like draw and hit elsewhere, but I think they'd be useful here as well.